modules/quality_control/jsx/columnFormatterImg.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Modify behaviour of specified column cells in the Data Table component
+ * @param {string} column - column name
+ * @param {string} cell - cell content
+ * @param {array} rowData - array of cell contents for a specific row
+ * @param {array} rowHeaders - array of table headers (column names)
+ * @return {*} a formated table cell for a given column
+ */
+function formatColumn(column, cell, rowData, rowHeaders) {
+  // If a column if set as hidden, don't display it
+  if (loris.hiddenHeaders.indexOf(column) > -1) {
```

Comment:
I don't see why this uses a global variable on the Loris object. If it's constant, it should just be inlined into the formatter, and if it's variable it should be a state on the React component that defines it which is passed around as a prop.